internal/pkg/agent/application/gateway/fleet/fleet_gateway.go

```diff
@@ -37,21 +37,24 @@ const (
 	fleetStateStarting = "starting"
 )
 
+// Default backoff settings for connecting to Fleet
+var defaultFleetBackoffSettings = backoffSettings{
+	Init: 60 * time.Second,
```

Comment:
The default of the initial enrollment during installation could be drastically slower because of this change. The original value for this was 1 second, so if it was just something like a network hickup or the server was very close to being ready it will now take 60 seconds versus the original 1 second.

Reply:
Hm, are confident we can't get a thundering herd problem with plain enroll? It can definitely happen with delayed.

The 60s initial delay primarily affects checkins in fleet gateway, where it's not really a problem, but that may be too long of a wait for an interactive command.

Reply:
I think it would be possible to get a thundering herd problem if Elastic Agent is being deployed out to a fleet of machines through some external orchestration platform.

It was more of a highlight of the fact that for an interactive command that is too long for the initial delay.

Reply:
Do you think it'd be ok to use the same initial delay (60s) for delayed enroll as we do for Fleet checkins, and set a shorter initial time (let's say 5s) while also adding jitter for the plain enroll command?

Reply:
Done.

Reply:
Thanks.